packages/ui-library/src/components/navigation/tab-bar/index.test.ts

```diff
@@ -45,18 +45,27 @@ describe('blr-tab-bar', () => {
   it('has a size md by default', async () => {
     const element = await fixture(BlrTabBarRenderFunction(sampleParams, tabsAsChildren));
 
-    const input = querySelectorDeep('.blr-tab-bar-group', element.getRootNode() as HTMLElement);
-    const className = input?.className;
+    const tabBar = querySelectorDeep('.blr-tab-bar-group', element.getRootNode() as HTMLElement);
+    const className = tabBar?.className;
 
     expect(className).to.contain('md');
   });
 
   it('has a size sm when "size" is set to "sm" ', async () => {
     const element = await fixture(BlrTabBarRenderFunction({ ...sampleParams, size: 'sm' }, tabsAsChildren));
 
-    const input = querySelectorDeep('.blr-tab-bar-group', element.getRootNode() as HTMLElement);
-    const className = input?.className;
+    const tabBar = querySelectorDeep('.blr-tab-bar-group', element.getRootNode() as HTMLElement);
+    const className = tabBar?.className;
 
     expect(className).to.contain('sm');
   });
+
+  it('is rendering tabs inside slot', async () => {
+    const element = await fixture(BlrTabBarRenderFunction({ ...sampleParams, size: 'sm' }, tabsAsChildren));
+
+    const tabBar = querySelectorDeep('.blr-tab-bar-group', element.getRootNode() as HTMLElement);
+    const tabs = querySelectorAllDeep('.nav-item-container', tabBar?.getRootNode() as HTMLElement);
+
+    expect(tabs).to.be.lengthOf(11);
```

Comment:
Instead of giving a hardcoded value, it might be more readable to use tabsAsChildren.length.